data/blog/introduction-to-glassmorphism-on-android.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Introduction to Glassmorphism on Android
+date: '2023-05-16'
+tags: ['ANDROID', 'KOTLIN', 'DESIGN', 'BEGINNER']
+draft: false
+summary: An introduction to glassmorphism using BlurView library
+images: []
+layout: PostLayout
+canonicalUrl:
+authors: ['othimar']
+---
+
+# Overview
+
+<TOCInline toc={props.toc} exclude="Overview" toHeading={3} />
+
+# What is glassmorphism?
+
+Glassmorphism is a design style that uses transparency and blur to create glass-like effect . For transparency, we can use a transparent color. For the blur effect, we will use **[BlurView](https://github.com/Dimezis/BlurView)** library.
+In this tutorial, we are going to create a transparent card-like View with a glasmorphism effect.
+
+![Final Result](/static/images/posts/introduction-to-glassmorphism-on-android/screenshot.png)
+
+I code in Kotlin but it's easily adaptable to Java.
+I'm using Intellij IDEA 2023.1. But everything should work fine with with any recent version of Intellij or Android Studio.
+The source code is available on [github](https://github.com/othimar/android-glassmorphism-sample-app).
+
+# Setup the project
+
+We will start with a basic activity project.
+
+## Add BlurView dependency
+
+First we add jitpack repository in **settings.gradle**,
+inside **dependencyResolutionManagement**.
+
+```kts
+repositories {
+    google()
+    mavenCentral()
+    maven { url 'https://jitpack.io' }
+}
+```
+
+Then add **BlurView** inside the file **build.gradle**, in the **app** folder.
+
+```gradle
+implementation 'com.github.Dimezis:BlurView:version-2.0.3'
+```
+
+Now, we can start coding.
+
+## let's code
```

Comment:
This headline is not necessary. You can directly put the next headline `## Add a background to the layout`